README.md

```diff
@@ -27,76 +27,40 @@ For a real life example of an ABCI application you can checkout [Ethermint](http
 
 
 ## Dependencies
-Make sure that you have Rust installed. The easiest way is to follow the instructions on [rustup](https://rustup.rs/).
+Make sure that you have Rust and Cargo installed. The easiest way is to follow the instructions on [rustup](https://rustup.rs/).
 
 
 ## Installation
 To test the examples, please clone this repository.
 ```bash
 git clone git@github.com:tendermint/rust-abci.git
 ```
-Please have a look at the dummy or counter app inside `src/bin` to get a feeling for how this server works.
+The `empty_app` example, found under the `examples` folder, is a good demonstration/bare minimum foundation for a Rust ABCI app.
 
 To use this library to build your own ABCI apps in rust you have to include the following in your `Cargo.toml` file.
-```bash
-[dependencies]
-rust-abci = "0.1.0"
-```
-
-Or if you want to live on the edge then point directly towards our git repo.
-```bash
+```toml
 [dependencies]
-rust-abci = { git = "git@github.com:tendermint/rust-abci.git" }
+abci = "0.3.0"
 ```
 
-
-----
-
-
 ## Running the examples
 
 ### Tendermint
-To run either of the example apps you have to have Tendermint installed and initialised. Please install it according to these [instructions](https://github.com/tendermint/tendermint). You can also check out [Ethermint](https://github.com/tendermint/ethermint/tree/develop) for an in-depth explanation of how ABCI apps work.
-Rememeber to run tendermint with the `--abci grpc` flag like so.
-```bash
-tendermint node --abci grpc
-```
-
-### Examples
-Once the Tendermint instance is up and running you can start either of the examples like this. Of course, please rememeber to
-switch into the rust-abci folder.
+To run either of the example apps you have to have Tendermint installed and initialised. Please install it according to these [instructions](https://github.com/tendermint/tendermint). After initializing and configuring the node, Tendermint can be run with:
 ```bash
-cargo run --features "grpc_support" --bin dummy
-
-cargo run --features "grpc_support" --bin counter
+tendermint node
```

Comment:
You'd have to run `tendermint init` first.

Reply:
That's why the README says Tendermint needs to be 'installed and initialized'.